### PR TITLE
TRIVIAL: Fix grouping documentation

### PR DESCRIPTION
**Pull request by ivan-nejezchleb**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/391/ : SUCCESS in 1m 26s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/221/ : SUCCESS in 5s

File: docs/pivot_table_component.md [hunk 1]
```diff
@@ -171,11 +171,11 @@ const sortBy = [
 
 ## Grouping
 
-By default, the same values in column attributes are grouped across the rows, which helps improve data readability in pivot tables. The header of the grouped columns is displayed at the top of the table even when you scroll through a long group.
+By default, the same row attribute values in column are grouped across the rows, which helps improve data readability in pivot tables. The header of the grouped columns is displayed at the top of the table even when you scroll through a long group.
```
Comment (evgeniabogranskaya) [inline docs/pivot_table_component.md:174]: By default, the same row attribute values in a column are grouped...

(the "a" article is added)
please add in all the affected versions of the topic

Comment (ivan-nejezchleb) [reply docs/pivot_table_component.md]: fixed

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/392/ : SUCCESS in 50s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/222/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/393/ : SUCCESS in 37s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/8/ : SUCCESS in 11s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/100/ : SUCCESS in 13s

**Pull request merged**